script/tidy

```diff
@@ -46,7 +46,7 @@ find . -name '*.tdy' -delete
 # .pc directory is used for "quilt" patch system (in Debian/Ubuntu)
 # it contains pre-patched files and should be ignored
 # shellcheck disable=SC2207
-files=($(find . ! -path '*/.pc/*' \( -name '*.p[lm]' -o -name '*.t' \)))
+files=($(find . ! -path '*/.pc/*' -type f \( -name '*.p[lm]' -o -name '*.t' \)) $(file --mime-type script/* | sed -n 's/^\(.*\):.*text\/x-perl.*$/\1/p'))
```

Comment:
This change is unrelated to the rest of the PR.

Reply:
Actually there is a clear relation: https://github.com/os-autoinst/openQA/pull/2137#issuecomment-507600748 explains that. However I can create another PR if this is what you prefer: https://github.com/os-autoinst/openQA/pull/2155